samples/bluetooth/fast_pair/locator_tag/sysbuild.cmake

```diff
@@ -16,3 +16,21 @@ if(NOT DEFINED FP_MODEL_ID AND NOT DEFINED FP_ANTI_SPOOFING_KEY)
   set(FP_MODEL_ID "0x4A436B" PARENT_SCOPE)
   set(FP_ANTI_SPOOFING_KEY "rie10A7ONqwd77VmkxGsblPUbMt384qjDgcEJ/ctT9Y=" PARENT_SCOPE)
 endif()
+
+if(SB_CONFIG_APP_DFU)
+  set_config_bool(${DEFAULT_IMAGE} CONFIG_APP_DFU y)
+
+  if(NOT SB_CONFIG_NETCORE_IMAGE_NAME STREQUAL "")
+    set(dfu_speedup_kconfig_fragment
+        ${APP_DIR}/sysbuild/${SB_CONFIG_NETCORE_IMAGE_NAME}/dfu_speedup_fragment.conf
+    )
+
+    if (EXISTS ${dfu_speedup_kconfig_fragment})
+      set(${SB_CONFIG_NETCORE_IMAGE_NAME}_EXTRA_CONF_FILE
```

Comment:
This CMake option may already exist. In this case, we must append our conf file instead of overwriting it.